tests/api/tasks/by_entities_uris.test.ts

```diff
@@ -1,60 +1,14 @@
 import 'should'
```

Comment:
When running these tests with empty databases (non-quick), I get the 3 following red tests:

```
  1) tasks:byEntitiesType
       should return tasks with a specific entities type:
     TypeError: Cannot read properties of undefined (reading 'entitiesType')
      at Context.<anonymous> (tests/api/tasks/by_entities_type.test.ts:34:14)
      at process.processTicksAndRejections (node:internal/process/task_queues:95:5)

  2) tasks:byEntitiesType
       should return a limited array of tasks:

      AssertionError: expected 0 to be 1
      + expected - actual

      -0
      +1
      
      at Assertion.fail (node_modules/.pnpm/should@13.2.3/node_modules/should/cjs/should.js:275:17)
      at Assertion.value (node_modules/.pnpm/should@13.2.3/node_modules/should/cjs/should.js:356:19)
      at Context.<anonymous> (tests/api/tasks/by_entities_type.test.ts:40:25)
      at process.processTicksAndRejections (node:internal/process/task_queues:95:5)

  3) tasks:byEntitiesType
       should take an offset parameter:
     TypeError: Cannot read properties of undefined (reading 'should')
      at Context.<anonymous> (tests/api/tasks/by_entities_type.test.ts:47:15)
      at process.processTicksAndRejections (node:internal/process/task_queues:95:5)

```

Reply:
Fixed in 7d0a14873 
Fixtures created human tasks instead instead of work tasks.

Reply:
One red test remaining (with or without empty databases):
```
    1) should return tasks with a specific entities type


  0 passing (199ms)
  1 failing

  1) tasks:byEntitiesType
       should return tasks with a specific entities type:
     TypeError: Cannot read properties of undefined (reading 'entitiesType')
      at Context.<anonymous> (tests/api/tasks/by_entities_type.test.ts:35:14)
      at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
```
The `tasks` array is empty

Reply:
Some `await` was missing,
should be fixed now in 2a80923ff